src/backend/joanie/settings.py

```diff
@@ -197,8 +197,12 @@ class Base(Configuration):
         "default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
     }
 
-    JOANIE_ANONYMOUS_COURSE_SERIALIZER_CACHE_TTL = 3600  # 1 hour
-    JOANIE_ENROLLMENT_GRADE_CACHE_TTL = 600  # 10 minutes
+    JOANIE_ANONYMOUS_COURSE_SERIALIZER_CACHE_TTL = values.IntegerValue(
```

Comment:
```suggestion
    JOANIE_ANONYMOUS_COURSE_SERIALIZER_CACHE_TTL = values.PositiveIntegerValue(
```
I know values.Value is even enough but as you've specified a kind, this one seems even more accurate

Reply:
can you actually use IntegerValue with None?

Reply:
Good idea, I'll use `PositiveIntegerValue`.
Firstly I used `IntegerValue` because `Value` interprets the given value as a string not a number.